haskell-src-meta/examples/HsHere.hs

```diff
@@ -42,7 +41,7 @@ data Here
   = CodeH Exp
   | TextH String
   | ManyH [Here]
-  deriving (Eq,Show,Data,Typeable)
```

Comment:
In 9.12, [`-Wderiving-typeable`](https://downloads.haskell.org/ghc/9.12.1/docs/users_guide/using-warnings.html#ghc-flag-Wderiving-typeable) has been added to `-Wall`.